clang/lib/CConv/CConv.cpp

```diff
@@ -168,7 +169,7 @@ void runSolver(ProgramInfo &Info,
 
   clock_t StartTime = clock();
   // FIXME: We should be seeing whether the constraints could be solved
```

Comment:
I think we can delete this FIXME comment? We are checking the solution now, right?